local_dev/pyproject.toml

```diff
@@ -6,7 +6,6 @@ readme = "README.md"
 requires-python = ">=3.13"
 dependencies = [
     "cdm-jupyterlab-brand-extension",
-    "cdm-minio-service-client",
```

Comment:
This was present in the pyproject.toml file but not in the requirements file that was being used to load python packages on the base image, so it wasn't getting installed on the docker image. I've therefore deleted it from this file.